internal/proto/controller/api/resources/accounts/v1/account.proto

```diff
@@ -41,23 +41,23 @@ message Account {
   ]; // @gotags: `class:"public"`
 
   // Output only. The time this resource was created.
-  google.protobuf.Timestamp created_time = 50 [json_name = "created_time"]; // @gotags: `class:"public"`
+  google.protobuf.Timestamp created_time = 50 [json_name = "created_time"]; // @gotags: `class:"public" eventstream:"observation"`
 
   // Output only. The time this resource was last updated.
-  google.protobuf.Timestamp updated_time = 60 [json_name = "updated_time"]; // @gotags: `class:"public"`
+  google.protobuf.Timestamp updated_time = 60 [json_name = "updated_time"]; // @gotags: `class:"public" eventstream:"observation"`
 
   // Version is used in mutation requests, after the initial creation, to ensure this resource has not changed.
   // The mutation will fail if the version does not match the latest known good version.
-  uint32 version = 70; // @gotags: `class:"public"`
+  uint32 version = 70; // @gotags: `class:"public" eventstream:"observation"`
```

Comment:
I've seen us skip the version in other PRs (and in other protos in this PR), is this correct?

Reply:
Based on my understanding this is not a product-type version and it is only being used for mutation so I did not include them in the previous proto and PR and I think I accidentally added here 😞

Reply:
@jimlambrt Please let me know what you thing about his version and adding to observation

Reply:
I don't think we need the resource versions.  I think, I've seen them included in other PRs.

Reply:
If we're not going to emit them in the observation I see some re-PRs of previous protos in our future.